persister/whisper.go

```diff
@@ -200,6 +234,7 @@ func (p *Whisper) updateMany(w *whisper.Whisper, path string, points []*whisper.
 	}()
 
 	// start = time.Now()
+	p.registerOutOfOrderLags(points)
```

Comment:
is it good idea to do that syncronously? maybe better put to goroutine?

Reply:
I did not notice any performance impact during testing, do you think we might have it in some cases?

Reply:
tested on the most loaded box, no issues there